Assets/Scripts/NPCs/FieldOfVision.cs

```diff
@@ -218,25 +218,26 @@ EdgeInfo FindEdge(ViewCastInfo minViewCast, ViewCastInfo MaxViewCast)
     ViewCastInfo ViewCast(float globalAngle)
 	{
         Vector3 direction = DirFromAngle(globalAngle, true);
+		print(direction);
```

Comment:
Can you get rid of this now that the issue is all fixed?

(I was wondering why I was only getting 15 FPS, then I saw that I had 999+ debug messages in the console.  I removed this line, and FPS went back up to ~150!)

Reply:
OOF big sads there, good catch 😨